src/api/config.ts

```diff
@@ -1,12 +1,19 @@
 import express from 'express';
 import { Config } from '../entities/config';
 import logger from '../logger';
+import { KnownConfig } from '../types/config';
 
 export const config = express.Router();
 
 config.get('/', async (req, res) => {
   try {
-    res.send(await Config.find({ order: { key: 'ASC' } }));
+    const configItems: Config[] = await Config.find();
+    Object.values(KnownConfig).forEach((knownConfigItem): void => {
+      if (!configItems.find((configItem: Config) => configItem.key === knownConfigItem)) {
+        configItems.push(new Config(knownConfigItem, 'undefined'));
```

Comment:
Will it let you put `null` as the value? I think a change that the UTD students did should allow that and a frontend change I have coming soon will show that properly.

```suggestion
        configItems.push(new Config(knownConfigItem, null));
```